_send_key.py

```diff
@@ -0,0 +1,22 @@
+from interception import  *
+from consts import *
+from stroke import *
+import time
+
+class ScanCode():
+    ESC = 1
+    M = 50
+    MINUS = 12
+
+    KEY_DOWN = 0
+    KEY_UP = 1
+
+def send_key(device:int, scancode:int):
+    c = interception()
+    c.send(device, key_stroke(scancode,ScanCode.KEY_DOWN,0))
+    c.send(device, key_stroke(scancode,ScanCode.KEY_UP,0))
+
+if __name__ == "__main__":
+    device = 8  # 0-10, 根据机器情况修改
```

Comment:
In my opinion, an example should be more generic, I guess that 8 corresponds to your keyboard, but it's not the case in every machine, using a loop to search for keyboards or simply get it as input seems more appropriate

Reply:
You are right, I do use loop detection in use, but I haven't updated the code yet